testing/adios2/bindings/C/TestBPWriteTypes.cpp

```diff
@@ -120,8 +120,16 @@ int main(int argc, char **argv)
     MPI_Init(nullptr, nullptr);
 #endif
 
-    ::testing::InitGoogleTest(&argc, argv);
-    int result = RUN_ALL_TESTS();
+    int result = -1;
+    try
```

Comment:
Why swallow the exception here rather than propagate it?

Reply:
Coverity is complaining about RUN_ALL_TEST not being caught.

Reply:
That's okay.  It's allowed to complain about some things.  Test in particular we want to crash if an exception makes it that fer since that means we didn't catch it internally where we should have.  I's rather keep the exception propagation here and teach coverity to ignore that specific issue

Reply:
Let's do the latter....I will change this back.

Reply:
It's part of getting used to both false alarms and acceptable errors with automated analysis tooling.  100% pass rate is never expected and in this case, even if we can't easily suppress it, I'd say it's an acceptable error.

Reply:
Contaminates the report.... let's get rid of this false positive.

Reply:
@chuckatkins so I just rerun the tests with intentional failures and the try-catch doesn't affect the end result. This is just catching an exception thrown by google-test itself, not the tests generated exceptions.

Reply:
Still though, we don't want to swallow exceptions, especially in test code.  If an exception get's thrown by the test framework then the test should crash.  Coverity flagging it as an issue is a generic "this could crash" type error, which in this case, is known and intended.

Reply:
I think we are going in circles here. If you can't provide an example on where this fails and if it can't be fixed with commenting out 2 lines of code then I don't think it's worth spending more time on this.